responses.go

```diff
@@ -222,9 +239,18 @@ type AccountResp struct {
 	CPULimit               AccountResourceLimit `json:"cpu_limit"`
 	Permissions            []Permission         `json:"permissions"`
 	TotalResources         TotalResources       `json:"total_resources"`
-	SelfDelegatedBandwidth DelegatedBandwidth   `json:"self_delegated_bandwidth"`
+	SelfDelegatedBandwidth *DelegatedBandwidth  `json:"self_delegated_bandwidth"`
 	RefundRequest          *RefundRequest       `json:"refund_request"`
 	VoterInfo              VoterInfo            `json:"voter_info"`
+
+	// added 2022-11-02
```

Comment:
We should actually write down in which version of EOSIO/LEAP this was actually added/changed rather than the date, as it can easily be found from a `git blame`.

Also, leaving a blank line before the fields so the comment is not attached to the field directly but is more informational in the code.

Please change it throughout the PR.

Reply:
Glad to hear you are active `git blame` user :) Let me track the version when it was changed

Reply:
https://github.com/eoscanada/eos-go/pull/191/commits/18d78b47dd7be8c506ce92f5422a4273eb0d1820